File: examples/aiogram-demo-app/bot.py
# Демо-бот на long polling: отвечает на /start и повторяет текстовые сообщения.
# Токен берётся из .env (BOT_TOKEN) — файл создаётся на вкладке «Переменные».
import asyncio
import os
import sys

from aiogram import Bot, Dispatcher
from aiogram.filters import CommandStart
from aiogram.types import Message
from dotenv import load_dotenv

load_dotenv()

token = os.getenv("BOT_TOKEN")
if not token:
    print("Не задан BOT_TOKEN. Добавьте его в .env рядом с bot.py.", file=sys.stderr)
    sys.exit(1)

dp = Dispatcher()


@dp.message(CommandStart())
async def start(message: Message) -> None:
    await message.answer("Бот работает. Отправьте любое сообщение — он повторит его.")


@dp.message()
async def echo(message: Message) -> None:
    await message.answer(message.text or "Это не текстовое сообщение.")


async def main() -> None:
    await dp.start_polling(Bot(token))


if __name__ == "__main__":
    asyncio.run(main())

File: examples/bot-demo-app/bot.js
// Демо-бот на long polling: отвечает на /start и повторяет текстовые сообщения.
// Токен берётся из .env (BOT_TOKEN) — файл создаётся на вкладке «Переменные».
require("dotenv/config");
const { Bot } = require("grammy");

if (!process.env.BOT_TOKEN) {
  console.error("Не задан BOT_TOKEN. Добавьте его в .env рядом с bot.js.");
  process.exit(1);
}

const bot = new Bot(process.env.BOT_TOKEN);

bot.command("start", (ctx) =>
  ctx.reply("Бот работает. Отправьте любое сообщение — он повторит его."),
);
bot.on("message:text", (ctx) => ctx.reply(ctx.msg.text));

bot.start();
